third_party/move/move-compiler-v2/tests/unit_test/test/attribute_location_invalid.exp

```diff
@@ -23,6 +23,3 @@ error: invalid attribute
    │       │
    │       Attribute 'expected_failure' is not expected with a struct
    │       Expected to be used with one of the following: function
-
-
-============ bytecode verification succeeded ========
```

Comment:
Does this mean we do not run those tests to an end any more? This is not what we want. This flag should not be on for all our existing tests which are effectively deactivated now whenever there is a warning.

Reply:
These tests yield errors.  We no longer run validation in the test if an error occurs.  This matches the full compiler behavior.

Reply:
I'm sorry but until now those tests run until bytecode verification, and now they do not any longer, if I read this here right. This significantly removes our test coverage. The compiler tests should either (a) not treat warnings as errors (except cases where we explicitly test for this, if needed -- can be done via a new test config) (b) fix all those tests and remove the warnings. I think (a) is more practical.

Reply:
No, they had errors and should have stopped before bytecode verification.  They only ran through bytecode verification because of a bug that didn't stop compilation before bytecode verification if there had been an error.  see move-compiler-v2/tests/testsuite.rs:682 (now `if ok && ...`).  I'm pretty sure that errors were missed because ok is just overwritten later.